starlite/openapi/spec/components.py

```diff
@@ -32,41 +32,41 @@ class Components(BaseSchemaObject):
     """
 
     schemas: dict[str, Schema] | None = None
-    """An object to hold reusable [Schema
-    Objects](https://spec.openapis.org/oas/v3.1.0#schemaObject)."""
+    """An object to hold reusable
+    `Schema Objects <https://spec.openapis.org/oas/v3.1.0#schemaObject>`_"""
```

Comment:
These line breaks are necessary for some reason. See: sphinx-doc/sphinx/issues/11246